claim/index.html

```diff
@@ -97,15 +97,15 @@ <h1>Claim HNS</h1>
   <p>This page explains how <b>github developers with over 15 followers on February 2019</b>, or in the <b>PGP WoT Strong Set</b> can claim HNS. Being able to claim does NOT imply that one is a "top open source developer", this system was optimized for a list of previously scrapeable keys (and could not be modified after the Handshake network launches without a hard fork).</p>
   <h2>System Setup</h2>
   <p>Please read through these instructions carefully, as using cryptographic blockchains are a bit unusual.</p>
-  <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
+  <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential libunbound-dev</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
```

Comment:
`libunbound` is optional - hsd can run without it. Although it may throw a warning during installation, it should still run using the javascript recursive resolver instead. Are you sure it is required for the claim process?

Reply:
Hard to say I'm sure since I only did it once; it's possible there was some other problem but it wasn't working before I did this but was after.

Reply:
Done